webpack.config.js

```diff
@@ -61,16 +61,20 @@ const optimizationDetective = ( env ) => {
 				patterns: [
 					{
 						from: `${ source }/dist/web-vitals.js`,
-						to: `${ destination }/web-vitals.js`,
+						to: `${ destination }/build/web-vitals.js`,
```

Comment:
Is this missing the flag to say it is minified?

```suggestion
						to: `${ destination }/build/web-vitals.js`,
						info: { minimized: true }`,
```

Reply:
Yes! I just realized this. I should probably make this change in the config for Web Worker Offloading which is copying the partytown files.